docs/packaging/advanced-config/local-repository.md

```diff
@@ -51,18 +51,33 @@ With the `.eopkg` files now present in the local repository, you can use them to
 
 Every time you run `go-task build-local`, all `.eopkg` files in the local repository will be re-indexed.
 
+### A streamlined local repository workflow for stack updates
+
+If you know ahead of time that you are going to need to put the package files from the current build into the local repository as dependencies for subsequent builds, you can use:
+
+```bash
+go-task build-localcp
+```
+
+The `build-localcp` task will:
+- build the current package against any existing packages in your local repository
+- copy the newly built package files to your local repository
+- re-index your local repository
+
+The `build-localcp` workflow can be very convenient when you are doing stack updates.
+
 ### Best practices when working with a solbuild local repository
 
 There are some important things to know when working with local repositories, as they may lead to issues later on.
 
-  - `solbuild` will use your version of a package from the local repository regardless of whether there's a higher release in the Solus repository. Therefore:
-  -  Only use `go-task build-local` when required
-  -  Remove old packages from the local repository when they are no longer needed, and rebuild its index with the following commands (this will remove all locally built eopkgs)
-  ```bash
-  go-task clean-local
-  go-task build-localindex
-  ```
-- If a package is already installed in the `solbuild` image, the release must be higher for it to be installed.
+- `solbuild` will use your version of a package from the local repository regardless of whether there's a higher release in the Solus repository. Therefore:
+  - Only use `go-task build-local` or `go-task build-localcp` when required
+  - Remove old packages from the local repository when they are no longer needed, and rebuild its index with the following command (this will remove all locally built eopkgs)
+    ```bash
+    go-task clean-local
+    ```
+  - Every time you run `go-task clean-local`, the (now empty) local repository will be re-indexed automatically. This is a convenience feature.
```

Comment:
I think we can remove the words "with the following command", and everything after it on that line. I'm also not sure about the second bullet point being separate. I feel like it should be a part of the one above it, since it's still talking about `go-task clean-local`.